### DISCUSSION TITLE
Ipsec ikev1 between mac and strongswan ( linux)  is not being initiated at all.

### DISCUSSION THREAD

**Discussion by gowdamanil** [strongswan/strongswan#2203] [Q&A]
Hello tobias, 

I have installed strongswan on Mac client using homebrew. 

brew install strongswan

After installation, I see that strongswan.conf is not at all installed.  Just ipsec.conf and ipsec.secrets were installed, I just configured and restarted strongswan. 

Results: IPSec initiation itself not started. 

Am using transport mode ikev1. 
This is for compatibilty purpose. 

Can u suggest more on this. 

Mac is client.

Regards, 
Anil Gowda M

**Comment by gowdamanil** [comment]
Another thing to mention is,  the same linux strongswan had started between linux and windows

**Comment by tobiasbrunner** [comment]
> After installation, I see that strongswan.conf is not at all installed.

Where did you look? Anyway, if doesn't exist, you can also just create it (can be empty).

> Results: IPSec initiation itself not started.

Not sure what you mean.
